### PR TITLE
GH-70: Add hyperlinks for mailing list subscription and archives

### LINKED ISSUE #70: Community page links for mailing list are not hyperlinks
This page: https://parquet.apache.org/community/ has urls links for subscribing to the mailing list and finding the icons, but they aren't hyperlinks

This causes a small barrier to participate with the parquet community

Screenshot 2024-07-17 at 5 11 42 AM

I plan to fix this

### PR DESCRIPTION
**Pull request by alamb**
Closes https://github.com/apache/parquet-site/issues/70

Before this change the rls are not actually links. This put up a (minor) barrier to participating in the parquet communtity

image

After this change they are:

Screenshot 2024-07-17 at 5 20 53 AM

### CODE REVIEW

File: hugo.toml [hunk 1]
```diff
@@ -146,10 +146,20 @@ enable = false
 [params.links]
 # End user relevant links. These will show up on left side of footer and in the community page if you have one.
 [[params.links.user]]
-  name = "User mailing list"
+  name = "Post to Mailing List"
   url = "mailto:dev@parquet.apache.org"
   icon = "fa fa-envelope"
-  desc = "Discussion and help from your fellow users. To subscribe please email dev-subscribe@parquet.apache.org. View the archive here: https://lists.apache.org/list.html?dev@parquet.apache.org."
```
Comment (alamb) [inline hugo.toml:152]: I could not figure out how to make the content of this config file render as links. If someone can figure that we could potentially leave it as one icon/entry and just linkify these

I actually like a separate entry for the mailing list archives

Comment (vinooganesh) [reply hugo.toml:152]: Hmm unfortunately we're kind of stuck with what docsy offers out of the box: https://github.com/google/docsy/blob/309da207e5deeda762348bfd0cc32367c5565a1e/layouts/partials/community_links.html#L29 which doesn't treat the `desc` or any of the content inside of it as link

**Review by Fokko — APPROVED** [review]

**Review by wgtmac — APPROVED** [review]

**Comment by wgtmac** [comment]
Thanks @alamb! Let me merge this.

**Comment by alamb** [comment]
Thank you @wgtmac  🙏

**Pull request merged**